components/automate-cli/cmd/chef-automate/backup.go

```diff
@@ -791,11 +791,22 @@ func runDeleteBackupCmd(cmd *cobra.Command, args []string) error {
 		return err
 
 	}
+	var newArgs []string
 
-	if !backupDeleteCmdFlags.yes {
+	for i := 0; i < len(args); i++ {
+		location := "/var/opt/chef-automate/backups/" + args[i]
```

Comment:
we should not hardcode this Path, This might be either the default path or the customer-configured one, so you need to read the path from the configuration.